app/lib/features/tasks/pages/task_list_details_page.dart

```diff
@@ -45,53 +50,86 @@ class _TaskListPageState extends ConsumerState<TaskListDetailPage> {
   AppBar _buildAppbar() {
     final tasklistLoader = ref.watch(taskListItemProvider(widget.taskListId));
     return tasklistLoader.when(
-      data: (tasklist) => AppBar(
-        title: SelectionArea(
-          child: GestureDetector(
-            onTap: () => showEditTaskListNameBottomSheet(
-              context: context,
-              ref: ref,
-              taskList: tasklist,
-              titleValue: tasklist.name(),
-            ),
-            child: Text(
-              key: TaskListDetailPage.taskListTitleKey,
-              tasklist.name(),
-              style: Theme.of(context).textTheme.titleMedium,
-            ),
-          ),
-        ),
-        actions: [
-          PopupMenuButton(
-            icon: const Icon(Icons.more_vert),
-            itemBuilder: (context) {
-              return [
-                PopupMenuItem(
-                  onTap: () => showEditDescriptionSheet(tasklist),
-                  child: Text(
-                    L10n.of(context).editDescription,
-                    style: Theme.of(context).textTheme.bodyMedium,
-                  ),
+      data: (tasklist) {
```

Comment:
Just added `ActerIconWidget` in app bar and manage it's edit right based on the membership.